### PR TITLE
[app_id.arb] changing more thingy

### PR DESCRIPTION
**Pull request by MeFinity**
finally something to replace subtitle

### CODE REVIEW

File: lib/l10n/app_id.arb [hunk 1]
```diff
@@ -445,7 +445,7 @@
   "settingsThumbnailShowTagIcon": "Tampilkan ikon label",
   "settingsThumbnailShowLocationIcon": "Tampilkan ikon lokasi",
   "settingsThumbnailShowMotionPhotoIcon": "Tampilkan ikon Foto bergerak",
-  "settingsThumbnailShowRating": "Tampilkan peringkat",
+  "settingsThumbnailShowRating": "Tampilkan ",
```
Comment (deckerst) [inline lib/l10n/app_id.arb:448]: Didn't you miss "nilai"?

File: lib/l10n/app_id.arb [hunk 2]
```diff
@@ -150,14 +150,14 @@
...
   "albumTierRegular": "Lainnya",
 
-  "storageVolumeDescriptionFallbackPrimary": "Penyimpanan internal",
-  "storageVolumeDescriptionFallbackNonPrimary": "kartu SD",
-  "rootDirectoryDescription": "direktori root",
-  "otherDirectoryDescription": "direktori “{name}”",
+  "storageVolumeDescriptionFallbackPrimary": "Penyimpanan Internal",
+  "storageVolumeDescriptionFallbackNonPrimary": "Kartu SD",
+  "rootDirectoryDescription": "Direktori root",
+  "otherDirectoryDescription": "Direktori “{name}”",
```
Comment (deckerst) [inline lib/l10n/app_id.arb:160]: This one (and the one above "rootDirectoryDescription") are used within the sentence "storageAccessDialogMessage", so i don't think adding an uppercase is appropriate. Are you sure of this change?

Comment (MeFinity) [reply lib/l10n/app_id.arb:160]: i see those kind of things almost everytime on apps,but if you dont want it i'll remove it

File: lib/l10n/app_id.arb (hunk 1 above)
Comment (MeFinity) [reply lib/l10n/app_id.arb:448]: my text editor decided to remove "subtitle" on every line,filled it again
guessed i missed one

**Pull request merged**